README.md

```diff
@@ -11,6 +11,20 @@ Technical details are explained [here](native/README.md).
 
 Please note that this project is unrelated to the [Envoy Proxy project](https://www.envoyproxy.io/).
 
+# Getting Started
+
+## Example Applications
+
+* Wiki Unblocked: a fork of the official Wikipedia app with Envoy support [Play Store](https://play.google.com/store/apps/details?id=org.greatfire.wikiunblocked) [F-Droid](https://f-droid.org/en/packages/org.greatfire.wikiunblocked.fdroid/) [Source](https://github.com/greatfire/apps-android-wikipedia-envoy)
```

Comment:
i've been working off of this branch on wiki unblocked: https://github.com/greatfire/apps-android-wikipedia-envoy/tree/wiki-unblocked